app/test_main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import app.main
+from app.main import can_access_google_page
+from unittest import mock
+
+
+def test_access_google_page_without_internet_connection() -> None:
```

Comment:
The test function name `test_access_google_page_without_internet_connection` is descriptive, but consider using a more concise name that directly reflects the test scenario, such as `test_no_internet_connection`.